test/unit/storage.test.js

```diff
@@ -0,0 +1,99 @@
+const config = require('../../app/config/storage')
```

Comment:
Maybe a space after this line?

Reply:
Not sure it needs one, they're both just consts imported to support the test.